.gitignore

```diff
@@ -1,4 +1,6 @@
-composer.lock
```

Comment:
this shouldn't be removed. instead it should be updated with `composer update <vendor>/<package>`